Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/MetsXmlElementAccess.java

```diff
@@ -135,11 +139,16 @@ private MetsXmlElementAccess(Mets mets) {
         Map<String, FileXmlElementAccess> divIDsToPhysicalDivisions = new HashMap<>();
         if (optionalPhysicalStructMap.isPresent()) {
             DivType div = optionalPhysicalStructMap.get().getDiv();
-            FileXmlElementAccess fileXmlElementAccess = new FileXmlElementAccess(div, mets, useXmlAttributeAccess);
+            Map<FileType, String> fileUseByFileCach = createFileUseByFileCache(mets);
```

Comment:
Typo? Instead of `fileUseByFileCach` maybe `fileUseByFileCache` is better?

Reply:
Yes. Thanks. Will fix that. Something bigger is wrong, though. Integration tests are failing and its not clear to me why yet.